### PR TITLE
refine keras to Cluster Serving guide

### PR DESCRIPTION
**Pull request by Litchilitchy**
refine the guide according to the comments.

### CODE REVIEW

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 1]
```diff
@@ -815,7 +212,23 @@
...
+   "metadata": {},
+   "source": [
+    "We config the model path in `config.yaml` to following (if you do not know what is `config.yaml`, check [Cluster Serving Configuration](https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ClusterServingGuide/ProgrammingGuide.md#2-configuration))\n",
+    "```\n",
+    "## Analytics-zoo Cluster Serving\n",
+    "\n",
+    "model:\n",
+    "  # model path must be provided\n",
+    "  path: /path/to/transfer_learning_mobilenetv2\n",
+    "```\n",
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:230]: Can we show this as code block?

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 2]
```diff
@@ -815,7 +212,23 @@
...
    "source": [
-    "Then, we config the model path in `config.yaml` (if you do not know what is `config.yaml`, check [Cluster Serving Configuration](https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ClusterServingGuide/ProgrammingGuide.md#2-configuration))\n",
+    "### Deploy Cluster Serving\n",
+    "After model prepared, we start to deploy it on Cluster Serving."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:222]: You need to first 
(1) Install Cluster Serving
(2) Copy the model in the model directory (show the output of `ls folder` as an example).

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 3]
```diff
@@ -815,7 +212,23 @@
...
+    "We config the model path in `config.yaml` to following (if you do not know what is `config.yaml`, check [Cluster Serving Configuration](https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ClusterServingGuide/ProgrammingGuide.md#2-configuration))\n",
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:223]: where is `config.yaml`?

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 4]
```diff
@@ -815,7 +212,23 @@
...
+    "```\n",
+    "## Analytics-zoo Cluster Serving\n",
+    "\n",
+    "model:\n",
+    "  # model path must be provided\n",
+    "  path: /path/to/transfer_learning_mobilenetv2\n",
+    "```\n",
+    "\n",
     "\n",
     "After configuration, start Cluster Serving by `cluster-serving-start` as mentioned in [Cluster Serving Programming Guide](https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ClusterServingGuide/ProgrammingGuide.md#3-launching-service)"
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:233]: show the command to stat cluster serving

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 5]
```diff
@@ -815,7 +212,23 @@
...
    ]
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:234]: Put below into a new section "Prediction using Cluster Serving"

**Comment by jason-dai** [comment]
You shown include all the steps in this guide; the goal is that the user can follow just this guide to complete the example without looking at other documents. You can refer to other documents only for additional details.

**Comment by Litchilitchy** [comment]
@jason-dai modified according to comments. And now all the things could be reproduced instead of just showing words of Cluster Serving before.

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 6]
```diff
@@ -802,36 +199,297 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "mkdir: cannot create directory ‘cluster-serving’: File exists\n",
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:266]: Show the result when `cluster-serving` folder does not already exist

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 7]
```diff
@@ -802,36 +199,297 @@
...
+   ],
+   "source": [
+    "! head config.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prediction using Cluster Serving\n",
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:394]: This section should be `Start Cluster Serving`

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 8]
```diff
@@ -802,36 +199,297 @@
...
+    "Before start Cluster Serving, make sure Flink cluster is start, if not, call following (check [Cluster Serving Installation Guide](https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ClusterServingGuide/ProgrammingGuide.md#1-installation) for detail)"
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:395]: We should make it clear that Cluster Serving would require a Flink cluster, and refer the user to Flink deployment guide for more details (maybe in the beginning of the previous section).

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 9]
```diff
@@ -802,36 +199,297 @@
...
+     "output_type": "stream",
+     "text": [
+      "Starting cluster.\n",
+      "Starting standalonesession daemon on host Litchy-PC.\n",
+      "Starting taskexecutor daemon on host Litchy-PC.\n"
+     ]
+    }
+   ],
+   "source": [
+    "! $FLINK_HOME/bin/start-cluster.sh"
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:414]: Does this start a Flink cluster on the local node?

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 10]
```diff
@@ -802,36 +199,297 @@
...
+   ],
+   "source": [
+    "! cluster-serving-start"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we start Cluster Serving code"
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:476]: This should go into a new section `Predict using Cluster Serving`

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 11]
```diff
@@ -802,36 +199,297 @@
...
+      "SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]\n",
+      "log4j:WARN No appenders could be found for logger (org.apache.flink.client.cli.CliFrontend).\n",
+      "log4j:WARN Please initialize the log4j system properly.\n",
+      "log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.\n",
+      "[Full GC (Metadata GC Threshold)  32304K->20432K(1030144K), 0.0213821 secs]\n"
+     ]
+    }
+   ],
+   "source": [
+    "! cluster-serving-start"
```
Comment (jason-dai) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:469]: what's the Flink cluster address?

**Comment by jason-dai** [comment]
Replace 'Litchy' in the output to something generic (such as 'user')

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb (hunk 9 above)
Comment (Litchilitchy) [reply docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:414]: yes

**Comment by Litchilitchy** [comment]
@jason-dai all updated

**Pull request merged**